app/graphql/queries/sightings.rb

```diff
@@ -0,0 +1,11 @@
+module Queries
+  class Sightings < Queries::BaseQuery
+    description 'All sightings'
+
+    type [Types::SightingType], null: false
+
+    def resolve
+      ::Sighting.all
```

Comment:
I'm wondering what the difference here would be between `::Sighting.all` and `Sighting.all`?